test/particles.jl

```diff
@@ -1,15 +1,35 @@
 using QEDbase
+using StaticArrays
 using Random
 
+include("utils.jl")
+
 FERMION_STATES_GROUNDTRUTH_FACTORY = Dict(
     (Incoming, Electron) => IncomingFermionSpinor,
     (Outgoing, Electron) => OutgoingFermionSpinor,
     (Incoming, Positron) => IncomingAntiFermionSpinor,
     (Outgoing, Positron) => OutgoingAntiFermionSpinor,
 )
 
-rng = MersenneTwister(708583836976)
-x, y, z = rand(rng, 3)
+RNG = MersenneTwister(708583836976)
```

Comment:
Does it make sense to test more than one seed to discover precision issues?

Reply:
Good point! However, I think this is beyond the scope of this PR, because here we neither try to discover precision errors, nor to perform full-fledged fuzzy testing.
Nevertheless, both types of tests would be highly beneficial and will be addressed in a separate PR in the future (I opened #52 to remember that).

Reply:
A fixed seed can be a source for errors, because it is possible, that you "hit" a working case. But for the moment, it is okay fine.